jersey-servers/src/main/java/com/palantir/remoting3/servers/jersey/HttpRemotingJerseyFeature.java

```diff
@@ -61,6 +61,9 @@ public boolean configure(FeatureContext context) {
         // Tracing
         context.register(new TraceEnrichingFilter());
 
+        // Limits
+        context.register(new RequestLimitingInterceptor());
```

Comment:
So the only reason why people are going this high internally is because we have no limits. So, by releasing this we're basically guaranteed to break _someone_, but there's going to be no sensible limit, this'd just be a transition issue.

Rather than making this optional, would prefer making this be remoting4.